onadata/libs/serializers/dataview_serializer.py

```diff
@@ -199,7 +200,7 @@ def get_count(self, obj):
 
         return None
 
-    def get_last_submission_time(self, obj):
+    def get_last_submission_time(self, obj) -> Optional[str]:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/Ona/onadata/pullRequest?prid=5558448)